parts/linux/cloud-init/artifacts/labels.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Update Labels for Kubernetes nodes
```

Comment:
It's probably important to note that we never remove labels this way. Somethign we might need to remember in upgrade. Should we wipe labels right before reimaging?

Reply:
Oh, good point. I was thinking that the reimaged node always came up with blank labels.

This would only be an issue if we stop using one of the labels, for instance if down the line we decide that we are going to do reset SP in a different way and stop using the label than those labels would never be cleaned up, correct?

If we wipe it would we be wiping any custom labels customers put on the node? Not sure if that is the behavior we want.

Reply:
Customers can either add labels via kubectl, or on cluster creation with the label being preserved by this service. I just realized is that behavior we want our customers to experience. I feel like it would be unexpected to have labels I added during start up automatically be added back if I delete them. Seems like something we should document, or possibly bundle in an announcement somewhere?

Reply:
We could also possible give all the labels used in this way a prefix, i.e. ```preserve/<label>```. That way we would be able to clean up all of the labels with that prefix before reapplying labels, and it gives the customers a clear understanding of what labels are being handled by this service.

However, I'm guessing that would be be perfect as it would be a strange customer experience to also not get a label matching the exact value you used when creating the node.

Reply:
Thoughts?

Reply:
Chatted offline, the customer experience of having the labels come back is perfectly fine.

I don't see anything we need to change in the logic within AgentBaker at least in this iteration, so I simply changed the comment to:
```# Updates Labels for this Kubernetes node (adds any missing, updates others to newest values, but never removes labels)```